content/it/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,97 @@
+
+(In realtà c'è un _controller_ che scala orizzontalmente i nodi nel cluster. Vedi [Cluster autoscaling](/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaling)).
+
+## Stato desiderato versus corrente {#desired-vs-current}
+
+Kubernetes ha una visione *cloud-native* dei sistemi, ed è in grado di gestire continue modifiche.
+
+Il cluster viene modificato continuamente durante la sua attività ed il _circuito di controllo_ è in grado di risolvere automaticamente i possibili guasti.
+
+Fino a che i _controllers_ del cluster sono in funzione ed in grado di apportare le dovute modifiche, non è rilevante che lo stato complessivo del cluster sia o meno stabile.
+
+## Progettazione
+
+Come cardine della sua progettazione, Kubernetes usa vari _controllers_ ognuno dei quali è responsabile per un particolare aspetto dello stato del cluster. Più comunemente, un dato _circuito di controllo_ (_controller_) usa un tipo di risorsa per il suo stato desiderato, ed utilizza anche risorse di altro tipo per raggiungere questo stato desiderato. Per esempio il Job _controller_ tiene traccia degli oggetti di tipo _Job_ (per scoprire nuove attività da eseguire) e degli oggetti di tipo _Pod_ (questi ultimi usati per eseguire i _Jobs_, e quindi per controllare quando il loro lavoro è terminato). In questo caso, qualcos'altro crea i _Jobs_, mentre il _Job_ _controller_ crea i _Pods_.
+
+È utile avere semplici _controllers_ piuttosto che un unico, monolitico, _circuito di controllo_. I _controllers_ possono guastarsi, quindi Kubernetes è stato disegnato per gestire questa eventualità.
+
+{{< note >}}
+Ci possono essere diversi _controllers_ che creato o aggiornano lo stesso tipo di oggetti. Dietro le quinte, i _controllers_ di Kubernetes si preoccupano esclusivamente delle risorse (di altro tipo) collegate alla risorsa primaria da essi controllata.
+
+Per esempio, si possono avere _Deployments_ e _Jobs_; entrambe creano _Pods_. Il Job _controller_ non distrugge i _Pods_ creati da un _Deployment_, perché ci sono informazioni (*{{< glossary_tooltip term_id="label" text="labels" >}}*) che vengono usate dal _controller_ per distinguere i _Pods_.
+{{< /note >}}
+
+## I modi per eseguire i _controllers_ {#running-controllers}
+
+Kubernetes annovera un insieme di _controllers_ nativi che sono in esecuzione all'interno del {{< glossary_tooltip term_id="kube-controller-manager" >}}. Questi _controllers_ nativi forniscono importanti funzionalità di base.
+
+Il Deployment _controller_ ed il Job _controller_ sono esempi di _controllers_ che vengono forniti direttamente da Kubernetes stesso (ovvero _controllers_ "nativi").
+Kubernetes consente di eseguire un _control plane_ resiliente, di modo che se un dei _controllers_ nativi dovesse fallire, un'altra parte del _control plane_ si occuperà di eseguire quel lavoro.
```

Comment:
Fine, no problem. I thought words/expressions that are part of k8s lingo should not being translate, but if you think is better so, no problem, I'll fix it. Should I change "control plane" w/ "piano di controllo" anywhere in page/glossary?

Reply:
I'm -1 to use "piano di controllo"
"Control plane" (or "control loop") are core concepts of Kubernetes and those terms have very specific meanings. IMO translating those terms might create confusion

Reply:
To be clear: these are information technology concepts that existed before Kubernetes; in fact, control loops existed before computers were widespread. https://it.wikipedia.org/wiki/Controllore_logico_programmabile#Schede_PID_(Proporzionale_Integrale_Derivativo) uses Italian terms such as “controllori” almost exclusively because that follows the Wikipedia style guide.

_However_ if Italian readers are accustomed to seeing English jargon mixed in, there's not reason to translate everything here. I do like how https://kubernetes.io/pt/docs/concepts/architecture/controller/ provides Portuguese glosses for English terms on their first use. Maybe do that?

Reply:
Hi all. I'm a bit confuse about how to proceed. I understood the @sftim point to translate all the world/expression eve if they are part of k8s lingo to be compliant w/ others lan versions (eg. control loop e control plane will be translate including the first time the eg word in parenthesis, while we are not translating "controller"). On the other hand it seems italian k8s community seems more in line w/ @fabriziopandini proposal (https://t.me/kubernetes_it/30623). Please help me on this so I could finalize this issue and use it as guidelines for the next translation. Thanks a lot for you support

Reply:
I don't speak Italian well, so please dilute my opinions. However, I _recommend_ that you consider that your audience is people who mainly don't already understand Kubernetes.

Reply:
Thanks @sftim. I agree on that motivation, I'll try to stick as mach as I could w/ your and Portuguese approach: k8s lingo will be translated adding in parenthesis the en version for the first occurrence.